bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -744,15 +749,19 @@ spec:
                 - /var/run/ovn/ovnsb_db.ctl
                 - exit
         readinessProbe:
-          initialDelaySeconds: 60
+          initialDelaySeconds: 90
           timeoutSeconds: 5
           exec:
             command:
             - /bin/bash
             - -c
             - |
-              set -xe
```

Comment:
We should probably have `set  -eo pipefail`, so if ovn-appctl fails, we still fail the readiness check.

Reply:
@squeed It will fail because in case of ovn-appctl failure, leader_status will be empty as well.

Reply:
@squat i take that back. You are right, check will pass and readiness probe will be success. I removed it because if grep doesn't match to any string, it return exit code 1, and that actually was failing the readiness probe as well, which it shouldn't.

Reply:
cc @squeed